### DISCUSSION TITLE
Why it's not recommended ? CC2530 & CC2531

### DISCUSSION THREAD

**Discussion by ch4r13** [Koenkk/zigbee2mqtt#8441] [Devices]
Hello, I would like to ask why these two devices are not recommended. 
I'm having a few problems with HomeAssistant, but I'm not sure if that could be the reason :).
I mainly use the CC2531 because it's easy for me, just flash it, connect it to the Rpi and it works. Unless HomeAssistant is crashing ...
CC2530
Texas Instruments CC2531

**Comment by DubhAd** [comment]
> Adapters based on the CC2530 or CC2531 chip are not powerful and not recommended for networks larger than 20 devices.

Referencing [this article](https://electrolama.com/radio-docs/advanced/ti-part-numbers/) those sticks are using the oldest, least capable chips. You'll find posts on various forums, here (including in the issues), and many Discords from people who've found the sticks to become unreliable as the mesh size grows or it becomes busy.

My own experience is that the stick will become unresponsive at random. It might be fine for many days, or even weeks, and then it just stops responding. Restart Z2M and _mostly_ things would work, except for the times that the stick needed to be power cycled.
